crates/cheatcodes/src/fs.rs

```diff
@@ -250,6 +250,28 @@ impl Cheatcode for writeLineCall {
     }
 }
 
+impl Cheatcode for getArtifactPathCall {
+    fn apply(&self, state: &mut Cheatcodes) -> Result {
+        let Self { contractName: name } = self;
+        let path = state
+            .config
+            .available_artifacts
+            .as_ref()
+            .and_then(|artifacts| {
+                artifacts
+                    .iter()
+                    .find(|(id, _)| {
+                        let id_name = id.name.split('.').next().unwrap();
```

Comment:
why do we need this?
isn't this the contract name already?

Reply:
ArtifactId.name could have the form `Counter.0.8.23`